interpreter/text/arrange.ml

```diff
@@ -359,12 +359,20 @@ let data i seg =
   Node ("data $" ^ nat i, segment_mode "memory" dmode @ break_bytes dinit)
 
 
+(* Events *)
+
+let event off i e =
+  Node ("event",
```

Comment:
This could insert a symbolic name for better readability, analogous to the other forms of definition.

Reply:
Done